draft-packages/avatar/KaizenDraft/Avatar/Avatar.tsx

```diff
@@ -98,21 +111,24 @@ export const Avatar = ({
       {avatarState !== "none" && (
         <img
           ref={image}
-          className={styles.avatarImage}
+          className={cx(styles.avatarImage, {
+            [styles.companyAvatarImage]: isCompany,
+          })}
           src={avatarSrc}
           onError={onImageFailure}
           onLoad={onImageSuccess}
-          alt={fullName}
+          alt={fullName || " "}
```

Comment:
In terms of accessibility - Wouldn't this need to be `"`" or does `" "` do the same thing? Question for @dougmacknz

Reply:
Ooh yeah good catch!